### PR TITLE
Input

### PR DESCRIPTION
**Pull request by ikkurthis1998**
https://user-images.githubusercontent.com/71547157/154792692-ecc54d80-3966-4f4b-8cf1-e4216b907313.mov

### CODE REVIEW

File: pages/input.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			<nav class="nav">
+				<a class="nav-logo" href="../index.html">
+					<img
+						src="../images/ueye_logo.png"
+						alt="Ueye Logo"
+						class="nav-logo-image"
+					/>
+					<!-- <a>
+						<h1 class="nav-title">Ueye</h1>
+					</a> -->
```
Comment (jeetsdev) [inline pages/input.html:27]: Please remove unnecessary code.

**Review by jeetsdev — APPROVED** [review]
Looks good 👍🏻

**Review by saketkum11 — APPROVED** [review]
Good to go Ahead

**Review by ayush65 — APPROVED** [review]

File: pages/input.html (hunk 1 above)
Comment (ikkurthis1998) [reply pages/input.html:27]: I am yet to decide if I have to go with just an image as logo or use image + name as logo, will remove it once I decide.

Comment (jeetsdev) [reply pages/input.html:27]: Got it.

**Pull request merged**